packages/quantic/force-app/main/default/lwc/quanticCaseClassification/quanticCaseClassification.css

```diff
@@ -0,0 +1,37 @@
+lightning-icon{
+  --sds-c-icon-color-foreground-default: white;
+}
+.slds-visual-picker__figure{
+  width: 9.875rem;
+  height: 1.938rem;
+}
+
+.loading-holder{
```

Comment:
maybe linter needs to run on this file to have the `{` with a space after the class?